tests/test_game.py

```diff
@@ -23,3 +23,71 @@ def test_game_from_stock(self):
         self.game.deal_from_stock()
         self.assertEqual(len(self.game.waste), prev_waste_len + 1)
         self.assertEqual(len(self.game.stock), prev_stock_len - 1)
+
+    def test_move_tableau_pile(self):
+        # Setup: Ensure we have a known state
```

Comment:
can you update the comments in tests, using a structure given, when, then? and also adding spaces between them so that's easier to read ?